tests/BlazingSQLTest/EndToEndTests/windowFunctionTest.py

```diff
@@ -971,6 +969,120 @@ def executionTest():
                 fileSchemaType,
             )
 
+            queryId = "TEST_38"
+            query = """select first_value(c_nationkey) over 
```

Comment:
This test is a little trivial, because you are asking for the first value of c_nationkey, but you are partitioning by c_nationkey, therefore the first_value in that partition is the same as every value in the partition. Its an interesting edge case, but its not a good test for `first_value`

Reply:
Please add another similar test case, where you are getting the first value of something that you are not partitioning by

Reply:
Another issue with these test queries is that you have an order by at the end, which means that the order that was part of the windowing gets overridden by the order by at the end. Please take out the order by at the end to make sure the order we end up with at the end of the window function is correct